xontrib/vox.py

```diff
@@ -1,78 +1,102 @@
 """Python virtual environment manager for xonsh."""
 
 import sys as _sys
+import argparse as _argparse
 import xontrib.voxapi as _voxapi
 
 
 class _VoxHandler:
     """Vox is a virtual environment manager for xonsh."""
+    parser = _argparse.ArgumentParser(prog='vox', description=__doc__)
+    subparsers = parser.add_subparsers(dest='command')
+
+    create = subparsers.add_parser('new', aliases=['create'],
+        help='Create a new virtual environment')
```

Comment:
flake8 complaint here, too, that the line is under-indented.  if you have `help` directly under the first `'` it should be fine.